main.tf

```diff
@@ -96,6 +96,8 @@ locals {
   stage       = local.formatted_labels["stage"]
   name        = local.formatted_labels["name"]
 
+  root_module = var.root_module_tag_enabled ? basename(path.cwd) : null
```

Comment:
I found that using this with terragrunt doesn't have the intended output. It will output the temporary directory name from within the `.terragrunt-cache/` dir. This might be relevant because this module is used in lots of scenarios, not just within the atmos workflow.

Reply:
@joe-niland didn't know you were using Terragrunt! Good to know -- may need to ping you at some point in the coming weeks 😁